src/person.h

```diff
@@ -157,6 +157,12 @@ class Person : public VoxelObject {
 	uint8 GetInparkDirections();
 
 	virtual void DecideMoveDirection() = 0;
+	void DecideMoveDirectionOnPathlessLand(Voxel *former_voxel,
+	                                       const XYZPoint16 &former_vox_pos,
+	                                       const TileEdge exit_edge,
+	                                       const int dx,
+	                                       const int dy,
+	                                       const int dz);
```

Comment:
Don't think this is the code style we aim for, but I can see the use of it.

Reply:
I had initially made this function `virtual` which made the line unreasonably long, but now I collapsed it into the usual style again.